recipes/pycovjson/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pycovjson" %}
+{% set version = "0.3.9" %}
+{% set sha256 = "f1482516307189f8b7b8b0f3eeea645923298cea2bfb67f43f0d5e29fa2cc504" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
This package looks like a candidate for `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
ack, done